src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -70,21 +78,8 @@ def copy(
         )
 
     if is_get:
-        target = Path(destination_path).resolve()
-        cursor = StageManager().get(
-            stage_path=source_path, dest_path=target, parallel=parallel
-        )
-    else:
-        source = Path(source_path).resolve()
-        local_path = str(source) + "/*" if source.is_dir() else str(source)
-
-        cursor = StageManager().put(
-            local_path=local_path,
-            stage_path=destination_path,
-            overwrite=overwrite,
-            parallel=parallel,
-        )
-    return QueryResult(cursor)
+        return _get(recursive, source_path, destination_path, parallel)
+    return _put(recursive, source_path, destination_path, parallel, overwrite)
```

Comment:
Nit but let's use keywords arguments, it's easier to refactor later on

Reply:
Added